CONTRIBUTING.md

```diff
@@ -82,21 +124,28 @@ Or remove the ::test_mytest suffix to run all tests in the file:
 python3 -m pytest path_to_file
 ```
 
-- If using PyCharm, we highly recommend `pip install pytest-pycharm` to [improve ease of debugging inside of unit tests](https://stackoverflow.com/questions/14086067/debugging-pytest-post-mortem-exceptions-in-pycharm-pydev).
+- If using PyCharm, we highly recommend `pip install pytest-pycharm` to [improve ease of debugging inside unit tests](https://stackoverflow.com/questions/14086067/debugging-pytest-post-mortem-exceptions-in-pycharm-pydev).
 
 - To otherwise test your code changes, we recommend running AutoGluon on multiple datasets and verifying the code runs correctly and the resulting accuracy of the trained models is not harmed by your change.  One easy way to test is to simply modify the scripts in [`examples/`](https://github.com/autogluon/autogluon/tree/master/examples), or the [tutorial notebooks](https://github.com/autogluon/autogluon/tree/master/docs/tutorials), which already provide datasets.
 
 - Remember to update all existing examples/tutorials/documentation affected by your code changes.
 
-- We also encourage you to contribute new tutorials or example scripts using AutoGluon for applications you think other users will be interested in. Please see [`docs/tutorials/`](https://github.com/autogluon/autogluon/tree/master/docs/tutorials) or [`examples/`](https://github.com/autogluon/autogluon/tree/master/examples). All tutorials should be Jupyter notebooks converted into markdown (.md) files by running the command `jupyter nbconvert --ClearOutputPreprocessor.enabled=True --to markdown tutorial.ipynb`. This command also clears out any output cells as our build system will rebuild the .ipynb files from the markdown file and execute the notebooks rendering the output on our website. This is especially important for major new functionality. You can also directly edit .md files in a Jupyter notebook via these steps: https://d2l.ai/chapter_appendix-tools-for-deep-learning/jupyter.html#markdown-files-in-jupyter
-
-- After you open your pull request, our CI system will run for little while to check your code and report found errors. Please check back and fix any errors encountered at this stage (you can retrigger a new CI check by pushing updated code to the same PR in a new commit).
-
+- We also encourage you to contribute new tutorials or example scripts using AutoGluon for applications you think other users will be interested in. Please see [`docs/tutorials/`](https://github.com/autogluon/autogluon/tree/master/docs/tutorials) or [`examples/`](https://github.com/autogluon/autogluon/tree/master/examples). All tutorials should be Jupyter notebooks (.ipynb) files.
```

Comment:
Side note: Is our `examples` folder up-to-date / maintained?

Reply:
almost certainly not, will remove the mention here.